config/types/files.go

```diff
@@ -82,6 +84,13 @@ func init() {
 		r := report.Report{}
 		files_node, _ := getNodeChildPath(ast, "storage", "files")
 		for i, file := range in.Storage.Files {
+			if file.Mode == nil {
```

Comment:
why is this not in the validation logic?

Reply:
Moved the report logic to validation functions on `File` and `Directory`